examples/drupal_loadtest.rs

```diff
@@ -22,48 +22,48 @@
 //! See the License for the specific language governing permissions and
 //! limitations under the License.
 
+use goose::boilerplate;
+boilerplate!();
+
```

Comment:
The traditional way to write this is `use goose::prelude::*`

Reply:
We can add prelude in a followup PR